neurom/core/dataformat.py

```diff
@@ -43,15 +44,9 @@ class COLS:
     XYZR = slice(0, 4)
 
 
-class POINT_TYPE:
-    """Point types.
+Point = namedtuple('Point', ('x', 'y', 'z', 'r'))
 
-    These follow SWC specification.
-    """
-    (UNDEFINED, SOMA, AXON, BASAL_DENDRITE, APICAL_DENDRITE,
-     FORK_POINT, END_POINT, CUSTOM) = range(8)
```

Comment:
[not for this review]  Same for a enum like object factorization like this are so misleading ...